src/Components/Common/LabelForm/index.tsx

```diff
@@ -0,0 +1,40 @@
+import Heading from '@/Components/Heading';
+import { ErrorMsg } from '@/Pages/CreateRecruitment/page';
+import { RecruitFormWithSelect } from '@/Types/study';
+import { FieldErrors } from 'react-hook-form';
```

Comment:
```suggestion
import { FieldErrors, FieldValues } from 'react-hook-form';
```

범용 `FieldValues` 타입을 RHF으로부터 가져옵니다